gno.land/pkg/sdk/vm/keeper_test.go

```diff
@@ -115,6 +115,45 @@ func Echo(cur realm) string {
 	assert.Nil(t, memFile)
 }
 
+func TestVMKeeperAddPackage_DraftPackage(t *testing.T) {
```

Comment:
This test looks good. However, can you also verify that draft packages can be imported by other Genesis packages, but not by non-Genesis ones?